src/conversion/area/_area.js

```diff
@@ -0,0 +1,39 @@
+const shared = require('../shared');
```

Comment:
I, for one, would prefer a bit more descriptive filenames, something along the lines of `area_shared.js` or similar, but this is just me nit picking.

Reply:
Renamed to `shared_conversion_functions`

Reply:
Ah damn, I didn't mean to refer to that exact line lol. I was talking about the renaming the files `_area.js` to `area_shared.js`, `_velocity.js` to `shared_velocity.js`, the variable `shared` is completely fine and self explanatory 😛